blueprints/plugin_testing/list_all_resources_test-blueprint.yaml

```diff
@@ -0,0 +1,365 @@
+#Blueprint used for testing list operations for these node_types:
+#
+#* cloudify.openstack.nodes.Server
+#* cloudify.openstack.nodes.KeyPair
+#* cloudify.openstack.nodes.Subnet
+#* cloudify.openstack.nodes.SecurityGroup
+#* cloudify.openstack.nodes.Router
+#* cloudify.openstack.nodes.Port
+#* cloudify.openstack.nodes.Network
+#* cloudify.openstack.nodes.FloatingIP
+#* cloudify.openstack.nodes.Volume
+#* cloudify.openstack.nodes.ServerGroup
+#* cloudify.openstack.nodes.Image
+#* cloudify.openstack.nodes.Flavor
+#* cloudify.openstack.nodes.HostAggregate
+#* cloudify.openstack.nodes.User
+#* cloudify.openstack.nodes.Project
+#
+
+tosca_definitions_version: cloudify_dsl_1_3
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.2/types.yaml
+  - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-openstack-plugin/master/plugin.yaml
+
+inputs:
+  openstack_username:
+    type: string
+    default: { get_secret: keystone_username }
+
+  openstack_password:
+    type: string
+    default: { get_secret: keystone_password }
+
+  openstack_auth_url:
+    type: string
+    default: { get_secret: keystone_url }
+
+  openstack_region:
+    type: string
+    default: { get_secret: keystone_region }
+
+  openstack_project_id:
+    type: string
+    default: 'b5a748ef817c4333b48380a239374472'
```

Comment:
@kbijakowski We we do not use secret pair here ?

Reply:
DONE. I added getting keystone_tenant_id here. Only one problem here is that (if I am not wrong) tenant_id isn't natively added to Cloudify Manager secret store during installation (bootstrap). It (tenant_id) is required for neutron resources because of this: https://github.com/cloudify-cosmo/cloudify-openstack-plugin/issues/205.